hack/prune-junit-xml/prunexml_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2022 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Copyright 2022 The Kubernetes Authors.
+Started from https://github.com/dims/kubernetes/blob/d20df79545a41e499082f3786f9953d65eee450d/cmd/prune-junit-xml/prunexml_test.go
+*/
+
+package main
+
```

Comment:
suggestion: I love that we've added a test for this, but I worry that it won't be regularly run in CI. Would it be worth adding to this line?

https://github.com/cert-manager/cert-manager/blob/fda92d7c809f05ec182d0ddde38123d1c3306304/make/test.mk#L6

e.g.

```make
WHAT ?= ./pkg/... ./cmd/... ./internal/... ./test/... ./hack/prune-junit-xml/...
```